go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/qedus/nds
+
+require (
+	golang.org/x/net v0.0.0-20181023162649-9b4f9f5ad519
+	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f // indirect
```

Comment:
When I run `go mod init` in the `github.com/qedus/nds` directory the `golang.org/x/sync` requirement is not present. How did you create the module for it to be present? (This is my first attempt at playing with go modules.)

Reply:
I just followed these steps from the go mod [howto wiki](https://github.com/golang/go/wiki/Modules#how-to-define-a-module):

1.  `go mod init`
2. `go build ./...`
3. `go test ./...`
4. `go mod tidy`